data/typelibrary/io-1.0.0/typelib/revpi/IORevPiAIO.fbt

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FBType Name="IORevPiAIO" Comment="Analog I/O module for Revolution Pi by KUNBUS GmbH">
+	<Identification Standard="61499-2" Description="Copyright (c) 2024 Monika Wenger
&#10; 
&#10;This program and the accompanying materials are made
&#10;available under the terms of the Eclipse Public License 2.0
&#10;which is available at https://www.eclipse.org/legal/epl-2.0/
&#10;
&#10;SPDX-License-Identifier: EPL-2.0" >
```

Comment:
It seems that there are some strange characters in the Description field. Here in github they are shown with `^M`. Can you please check if you can get rid of these?